### DISCUSSION TITLE
AI監控碳排量

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#161] [General]
**新聞內容:**
Hoepner組成名為「GreenWatch」的團隊，運用 AI人工智慧來監控700家企業的發聲管道，包含網站、媒體等等。接著，他們查核該企業的碳排量，是否在發布永續行為宣稱後每年降低7%，以在 2050年達成淨零（Net Zero）目標。
根據GreenWatch調查成果發現，在他們所監控的企業中，有高達95%組織的永續聲明具漂綠嫌疑；在工業、非必須性消費行業中，有8成企業可能誇大其綠色行動；而在能源產業中，誇大或造假公司之永續作為的情形則不到一半。而以地域來看，在日本有84％的企業的永續宣稱可能有漂綠嫌疑；其次是在美國有75%的企業。
瑞士的蘇黎世大學和德國的紐倫堡大學皆採用名為「ClimateBert」的模型，來分析企業的「氣候相關財務揭露（Task Force on Climate-related Financial Disclosures，簡稱 TCFD）」，以確認企業是否僅揭露對自身有利的資訊。

**新聞心得:**
我覺得以人工智慧來監控企業實行永續責任是一個很好的應用，不僅可以揭露一些不實的報告，或是企業只顯示對自身有幫助的資料，透過各管道的監測，能讓民眾、政府看出隱藏的錯誤訊息。

**新聞來源:**
https://ubrand.udn.com/ubrand/story/12117/5993348

**Comment by pocper1** [comment]
我覺得用AI 來監控非常好! COP26 有說要減碳，若能透過AI 監控各國的碳排量，是一個不錯的方式

**Comment by briankuo123** [comment]
用AI來監控除了能24小時全天追蹤之外，還能避免出現人為上的影響或疏失。

**Comment by wh20502** [comment]
淨零碳排放量我覺得存在一定程度的難度(2050完成)我覺得會無限趨近0，但很難真的0，除非有生之年看到新的方法。

**Comment by ghost** [comment]
AI控制二氧化碳的排放量 對地球來說是件好事 但我比較怕AI控制到能源突然斷電只因為為了要控制CO2排放

**Comment by jack90325** [comment]
排碳量再不減少，人類就要滅亡了，這種監測絕對是必須的，抓到那些不斷排碳的大國

**Comment by LanningYang** [comment]
目前要0碳似乎還沒辦法 只能盡量往低碳方向前進

**Comment by lx3211** [comment]
AI不只在醫療上，在環境保護上也盡了很大的心力呢

**Comment by NcuJerryKao** [comment]
比請AI拍 諸如看見台灣的電影
效果更好

**Comment by lukiea** [comment]
讓AI 來監控人類這件事 ，或許是件方便的事，我們能做得只有接受，並且祈禱人類不會因此而習慣一切

**Comment by larry40305** [comment]
對AI監控碳排量進行深入研究，在現今時代已經無法避免了。
